platforms/ide/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/TestExecutionBuildConfigurationAction.java

```diff
@@ -137,7 +138,7 @@ private void configureTestTasksForTestDescriptors(Context context) {
                             filter.includeCommandLineTest(cls, method);
                         }
                     }
-                    Set<String> commandLineIncludePatterns = filter.getCommandLineIncludePatterns();
+                    SetProperty<String> commandLineIncludePatterns = filter.getCommandLineIncludePatterns();
```

Comment:
💅 Should this be?

```suggestion
                    Provider<Set<String>> commandLineIncludePatterns = filter.getCommandLineIncludePatterns();
```

Reply:
We append values to that property at line 142 and 144, so we jhave to use SetProperty here

Reply:
But we could also just do `filter.getCommandLineIncludePatterns(),addAll()` and `filter.getCommandLineIncludePatterns().add()` to be less confusing. So I changed that so it's done that way